src/app/zone-egresses/views/item/DetailView.vue

```diff
@@ -43,15 +43,24 @@
           </template>
         </KCard>
 
-        <div v-if="(data.zoneEgressInsight?.subscriptions ?? []).length > 0">
-          <h2>{{ t('zone-egresses.detail.subscriptions') }}</h2>
+        <template
+          v-for="subscriptions in [props.data.zoneEgressInsight?.subscriptions ?? []]"
+          :key="subscriptions"
+        >
+          <div
+            v-if="subscriptions.length > 0"
+          >
+            <h2>{{ t('zone-egresses.routes.item.subscriptions.title') }}</h2>
 
-          <KCard class="mt-4">
-            <template #body>
-              <SubscriptionList :subscriptions="data.zoneEgressInsight?.subscriptions ?? []" />
-            </template>
-          </KCard>
-        </div>
+            <KCard class="mt-4">
+              <template #body>
+                <SubscriptionList
+                  :subscriptions="subscriptions"
+                />
+              </template>
+            </KCard>
+          </div>
+        </template>
```

Comment:
I simplified this for Dataplane subscriptions in a previous PR, so I did this here also for zone'gresses